packages/backend/src/peripherals/database/OAppConfigurationRepository.ts

```diff
@@ -0,0 +1,70 @@
+import { Logger } from '@l2beat/backend-tools'
+import { ChainId } from '@lz/libs'
+import type { OAppConfigurationRow } from 'knex/types/tables'
+
+import { OAppConfiguration } from '../../tracking/domain/configuration'
+import { BaseRepository, CheckConvention } from './shared/BaseRepository'
+import { Database } from './shared/Database'
+
+export interface OAppConfigurationRecord {
+  oAppId: number
+  targetChainId: ChainId
+  configuration: OAppConfiguration
+}
+
+export class OAppConfigurationRepository extends BaseRepository {
+  constructor(database: Database, logger: Logger) {
+    super(database, logger)
+    this.autoWrap<CheckConvention<OAppConfigurationRepository>>(this)
+  }
+
+  public async addMany(records: OAppConfigurationRecord[]): Promise<number> {
+    const rows = records.map(toRow)
+    const knex = await this.knex()
+
+    await knex('oapp_configuration').delete()
+    await knex('oapp_configuration').insert(rows)
+
+    return rows.length
+  }
+
+  public async findAll(): Promise<OAppConfigurationRecord[]> {
```

Comment:
```suggestion
  public async getAll(): Promise<OAppConfigurationRecord[]> {
```

We follow this convention that `find...` repository method returns one element, but `get...` returns multiple.